static/app/views/codecov/tests/onboardingSteps/onboardingStep.tsx

```diff
@@ -17,13 +24,53 @@ const Header = styled('h3')`
   line-height: 31px;
 `;
 
-// currently no styles are added but this is here for organization and future use
-function Content(props: React.ComponentProps<'div'>) {
-  return <div {...props} />;
+const Content = styled('div')`
+  margin-top: ${p => p.theme.space.xs};
+`;
+
+function ExpandableDropdown({
+  triggerContent,
+  children,
+}: {
+  children: React.ReactNode;
+  triggerContent: React.ReactNode;
+}) {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  return (
+    <ExpandableContentContainer>
+      <ExpandableContentTrigger>
+        {triggerContent}
+        <IconChevron
+          direction={isExpanded ? 'up' : 'down'}
+          onClick={() => setIsExpanded(!isExpanded)}
+        />
+      </ExpandableContentTrigger>
+      {isExpanded && <ExpandedContent>{children}</ExpandedContent>}
+    </ExpandableContentContainer>
+  );
 }
 
+const ExpandableContentContainer = styled('div')`
+  border-top: 1px solid ${p => p.theme.border};
+  margin-top: auto;
+  padding: ${p => p.theme.space.xl} ${p => p.theme.space['3xl']};
+`;
+
+const ExpandableContentTrigger = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const ExpandedContent = styled('div')`
```

Comment:
this can be `<Container padding="2xl, 0 0 0">` imported from `sentry/components/core/layout`